### PR TITLE
Setup React Book store application

### PR DESCRIPTION
**Pull request by GutemaG**
## Setup React Book store application
- Initialize project with components with ```create-react-app```
- Set linters
- Create two pages in ```src/pages Books and Categories```
- Set react router for two pages
- Add some Style to components

### CODE REVIEW

File: src/components/Book.jsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+        <h3>{title}</h3>
+        <p>{author}</p>
+      </div>
+      <div className="book-progress">
+        <h1>Book Progress</h1>
+      </div>
+    </div>
+  );
+};
+
```
Comment (MaganaAsiati) [inline src/components/Book.jsx]: - Kindly make sure that you implement the point I list below, as it is shown in the picture below  your book, doesn't have the button to remove please add it as required in the project :+1: 

> * Book
>   * Takes the book data as a prop and renders the title and the author
>   * Has a button for removing books
>     * the button wouldn't remove the books *yet*, this is just a placeholder for now
> 

image

File: src/components/NewBook.jsx [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+        <option value="action">Action</option>
+        <option value="science-fiction">Science Fiction</option>
+        <option value="economy">Economy</option>
+      </select>
+      <button type="submit" className="btn submit-btn">Add Book</button>
+    </form>
+  </div>
+);
+
+export default NewBook;
```
Comment (MaganaAsiati) [inline src/components/NewBook.jsx]: - [ ] You did a great job by adding the input for adding title to your book but I suggest you also add the input for the author and make sure that those inputs are editable. I simply explain it below :+1: :+1: 

> CreateNewBook
> 
> * Renders a form with inputs for the title and the author of the book
> * Has a button to submit
>   * the button wouldn't add the book *yet*, this is just a placeholder for now
> 

image

**Review by MaganaAsiati — CHANGES_REQUESTED** [review]
# STATUS: Required Changes ♻️♻️♻️ 
## Hi @GutemaG , I'm @MaganaAsiati, your reviewer for this PR  :wave: 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

so-close-gif-4

## Highlights

- Descriptive README file  :heavy_check_mark:👨🏻‍💻
- Great work using the right GitHub Flow.:heavy_check_mark:
- No Linter errors :heavy_check_mark:

## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._
- [ ] [OPTIONAL] Please make sure to make your commit messages more descriptive in the future. :heavy_check_mark:👨🏻‍💻
- [ ] [OPTIONAL]  kindly  Add a meaningful description to your pull request in the future which gives more information about what you implement in this milestone :heavy_check_mark:👨🏻‍💻


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Comment by GutemaG** [comment]
Thank you @MaganaAsiati for your descriptive comments. For next time I will do my best!

**Review by mckent05 — APPROVED** [review]
Hi @GutemaG, 

# STATUS: APPROVED ✔️ ✔️ 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations and well done! 🎉 
approved


## Highlights
- All linter checks are passing ✔️ ✔️ 
- Your PR is professional 👏 👏 
- Your project has been properly set up 👍 👍 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**